integrations/lambda/src/lambda_sample/lambda_function.py

```diff
@@ -0,0 +1,140 @@
+import json
+import os
+import boto3
+from botocore.exceptions import ClientError
+
+REGION_NAME = os.environ['REGION_NAME']
+DATABASE_NAME = os.environ['DATABASE_NAME']
+TABLE_NAME = os.environ['TABLE_NAME']
+BATCH_SIZE = int(os.environ['BATCH_SIZE'])
+MEM_STORE_RETENTION_PERIOD_IN_HOURS = int(os.environ['MEM_STORE_RETENTION_PERIOD_IN_HOURS'])
+MAG_STORE_RETENTION_PERIOD_IN_DAYS = int(os.environ['MAG_STORE_RETENTION_PERIOD_IN_DAYS'])
+
+# Initialize the Timestream client
+timestream_client = boto3.client('timestream-write', REGION_NAME)
+
+# Define your table retention properties
+RETENTION_PROPERTIES = {
+    'MemoryStoreRetentionPeriodInHours': MEM_STORE_RETENTION_PERIOD_IN_HOURS,
+    'MagneticStoreRetentionPeriodInDays': MAG_STORE_RETENTION_PERIOD_IN_DAYS
+}
+
+def create_timestream_database_and_table():
+    """Create Timestream database and table if they do not exist."""
+    try:
+        # Create database if it does not exist
+        timestream_client.create_database(DatabaseName=DATABASE_NAME)
+        print(f"Database '{DATABASE_NAME}' created successfully.")
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'ConflictException':
+            print(f"Database '{DATABASE_NAME}' already exists.")
+        else:
+            raise e
+
+    try:
+        # Create table if it does not exist
+        timestream_client.create_table(
```

Comment:
Customers can now define a partition key to use for newly-created Timestream for LiveAnalytics tables. It will default to `'MEASURE'` unless users specify a dimension to use.